### PR TITLE
chore: consistency

### PR DESCRIPTION
**Pull request by ShineyDev**
### Description
This PR adds consistency to files in the repository.

### Checklist

- [x] This PR makes changes to the code.
    - [ ] The changes are breaking.
    - [ ] The changes implement a new feature.
    - [ ] The changes fix an issue.
    - [ ] I updated the documentation to reflect these changes.
- [ ] This PR makes changes to the documentation.
    - [ ] The changes require a different version of sphinx.
    - [ ] I updated the configuration file to reflect these changes.

- [ ] This PR has been tested.

### CODE REVIEW

File: discord/ext/alternatives/command_suffix.py [hunk 1]
```diff
@@ -59,9 +57,9 @@ def valid(self):
     async def reinvoke(self, *, call_hooks=False, restart=True):
         if self.suffix is not None:
             # since the command was invoked with a suffix,
-            # we need to make sure the view doesn't try to skip a nonexistent prefix
+            # we need to make sure the view doesn"t try to skip a nonexistent prefix
```
Comment (SebbyLaw) [inline discord/ext/alternatives/command_suffix.py:60]: I don't think this should be changed

File: discord/ext/alternatives/specific_error_handler.py [hunk 2]
```diff
@@ -46,22 +49,24 @@ def decorator(func):
...
     try:
         other._handled_errors = self._handled_errors
     except AttributeError:
         pass
+
     return other
 
-Command._ensure_assignment_on_copy = _ensure_assignment_on_copy
\ No newline at end of file
+commands.Command._ensure_assignment_on_copy = _ensure_assignment_on_copy
```
Comment (NCPlayz) [inline discord/ext/alternatives/specific_error_handler.py:72]:
```suggestion
commands.Command._ensure_assignment_on_copy = _ensure_assignment_on_copy

```

File: discord/ext/alternatives/command_suffix.py (hunk 1 above)
Comment (NCPlayz) [inline discord/ext/alternatives/command_suffix.py:60]:
```suggestion
            # we need to make sure the view doesn't try to skip a nonexistent prefix
```

**Review by NCPlayz — COMMENTED** [review]
Looks alright so far.

File: discord/ext/alternatives/command_suffix.py (hunk 1 above)
Comment (ShineyDev) [reply discord/ext/alternatives/command_suffix.py:60]: @bijij see what you did.

Comment (ShineyDev) [reply discord/ext/alternatives/command_suffix.py:60]: @bijij see what you did.

**Review by NCPlayz — APPROVED** [review]
Thanks. I'll run some tests just in case there's any side effects with the import changes.

**Pull request merged**